ui-v2/app/services/services.js

```diff
@@ -21,6 +21,12 @@ export default Service.extend({
       .then(function(item) {
         const nodes = get(item, 'Nodes');
         const service = get(nodes, 'firstObject');
+        const tags = nodes
+          .reduce(function(prev, item) {
+            return prev.concat(get(item, 'Service.Tags') || []);
+          }, [])
+          .uniq();
```

Comment:
```js
const tags = nodes
  .reduce((all, item) => all.concat(get(item, 'Service.Tags') || []), [])
  .uniq()
```

I'm just gonna keep showing you arrow functions until you see the light 😉

Reply:
hehehe, right now I use them when I have reason to, I like your style though! :smile: